maven-scm-plugin/src/main/java/org/apache/maven/scm/plugin/AddMojo.java

```diff
@@ -54,7 +54,7 @@ public void execute() throws MojoExecutionException {
 
             checkResult(result);
 
-            getLog().info("" + result.getAddedFiles().size() + " files successfully added.");
+            getLog().debug("" + result.getAddedFiles().size() + " files successfully added.");
```

Comment:
This I would rather leave with info as this is an important hint as often usage of wildcards leads to unexpected files being added. I think one info per goal is not overly verbose.

Reply:
reverted